src/Views/AppInfoView.vala

```diff
@@ -1049,7 +1066,13 @@ public class AppCenter.Views.AppInfoView : AppCenter.AbstractAppContainer {
 
                         release_carousel.add (release_row);
 
+#if HAS_APPSTREAM_1_0
+                        var release_version = new AppStream.Relation ();
+                        release_version.set_version (release.get_version ());
+                        if (package.installed && release_version.version_compare (package.get_version ())) {
```

Comment:
I'm not sure if that's the correct way to handle this...

Reply:
I've left an alternative suggestion in a review comment for checking/testing